### PR TITLE
Add blog post about adopting OTel metrics in Cloud Foundry

### PR DESCRIPTION
**Pull request by ctlong**
This is our first time contributing to the OTel blog. Excited to hear your feedback!

---

**Preview**: https://deploy-preview-3466--opentelemetry.netlify.app/blog/2023/cloud-foundry/

### CODE REVIEW

File: content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md
Comment (chalin) [inline content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md:1]: Consider renaming the file simply `cloud-foundry.md` (unless you're planning to submit another blog post in 2023 :)).

**Comment by chalin** [comment]
Very cool! Welcome and thanks for the contribution. I'll drop a few initial copyedit comments shortly.

File: content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md
Comment (chalin) [inline content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md:2]: How about:

- Using "at" instead of "in" (unless you mean the platform, in which case consider adding "Platform"
- Consider dropping "Experience Report: " from the title and mentioning that in the introductory paragraph. This will make the title a little les wieldy. 

```suggestion
title: Adopting OpenTelemetry for Metrics at Cloud Foundry
```

File: content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md
Comment (chalin) [inline content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md:3]:
```suggestion
linkTitle: OpenTelemetry at Cloud Foundry
```

File: content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md
Comment (chalin) [inline content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md:8]:
```suggestion
```

File: content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md
Comment (chalin) [inline content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md:18]:
```suggestion
Cloud Foundry is a large multi-tenant platform as a service that runs 12-factor
```

**Review by chalin — COMMENTED** [review]
A few quick drive-by copyedit comments.

File: content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md
Comment (mkocher) [reply content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md]: We dropped experience report per your suggestion. We do mean the Cloud Foundry platform, which we usually refer to as just "Cloud Foundry" so I'm thinking we'll leave it that way. "in the Cloud Foundry platform" starts getting a little wordy

File: content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md
Comment (mkocher) [reply content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md]: we shortened this in the latest version, dropping 'Metrics' as you suggested but leaving it as 'in'

File: content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md
Comment (chalin) [reply content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md]: Sounds good. Thanks for the clarification, and I agree with not adding "platform".

I'm still seeing "Experience Report:" so maybe that edit is on it's way. (If you drop "Experience Report:" you don't need the quotes around the title anymore (FYI).

Comment (mkocher) [reply content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md]: That's right, @ctlong and I went back and forth on dropping it or keeping it in and it is still there for now.

We made an effort in our post to make it useful to anyone thinking about adopting open telemetry for metrics. It seemed like specifying that it's an experience report instead of an announcement might draw in more readers. WDYT?

Comment (chalin) [reply content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md]: Works for me! Let's keep it 👍🏻

**Review by cartersocha — APPROVED** [review]
Lgtm in general. We'll probably want to copy over the boiler plate integration heading from the Tyk blog cc @svrnm @austinlparker

File: content/en/blog/2023/cloud-foundry.md
Comment (chalin) [inline content/en/blog/2023/cloud-foundry.md:22]: This version of the alert has an **invalid link** and the markdown is **misformatted**. I've already applied copyedits to this version as it is found in #3431.

@svrnm @cartermp et al. I'd suggest that we:

- Work on the alert text through #3431 first. (Let's not do it in two PRs at the same time 🙃)
- After #3431 is merged, I can factor the note out into a reusable shortcode
- We can then include the shortcode here.

How does that sound?

**Review by chalin — CHANGES_REQUESTED** [review]

File: content/en/blog/2023/cloud-foundry.md
Comment (chalin) [reply content/en/blog/2023/cloud-foundry.md:22]: Ok, let's see what we can do.
Regardless, we'll need a @open-telemetry/blog-approvers to approve.
Maybe first, bring over the copyedits I made to the alert in #3431?

Comment (chalin) [reply content/en/blog/2023/cloud-foundry.md:22]: Also, you'll need to rebase:

> ⚠️ **This branch is out-of-date with the base branch**

Comment (ctlong) [reply content/en/blog/2023/cloud-foundry.md]: @chalin how does that look now with the changes?

Comment (chalin) [reply content/en/blog/2023/cloud-foundry.md]: @ctlong - the badge has landed. **Rebase** to have access to it. For an example of how to add the badge to your post, see `content/en/blog/2023/integrations.md` in:

- https://github.com/open-telemetry/opentelemetry.io/pull/3505/files

Essentially, just add `{{< blog/integration-badge >}}` at the top of your page.

File: content/en/blog/2023/cloud-foundry.md
Comment (chalin) [inline content/en/blog/2023/cloud-foundry.md:23]:
```suggestion
{{< blog/integration-badge >}}
```

**Comment by chalin** [comment]
Thanks for adding the badge.
I'll let @svrnm review the content as a blog approver.

@svrnm - On a technical note, we can merge this before 11/08, unless we revisit the date of the Integration blog to be whenever this post gets published.

File: content/en/blog/2023/cloud-foundry.md
Comment (svrnm) [inline content/en/blog/2023/cloud-foundry.md:4]:
```suggestion
date: 2023-11-08
```

**Review by svrnm — CHANGES_REQUESTED** [review]
Looks good to me overall, some suggestions to add links to different internal pages and one major change request around the decision process.

**Comment by cartersocha** [comment]
@ctlong remember to update the branch too!

**Review by svrnm — APPROVED** [review]
lgtm.

@ctlong you have disabled that maintainers can add commits to your PR, so please fix the formatting by running `npm run format` and push the updates to this

**Comment by svrnm** [comment]
@ctlong please update the branch once again such that we are able to merge

**Comment by ctlong** [comment]
> @ctlong please update the branch once again such that we are able to merge

@svrnm done!

> @ctlong you have disabled that maintainers can add commits to your PR.

My bad. I definitely didn't _intend_ to disable that feature. Now it appears I can't change it without recreating the PR. If we keep running into rebase problems I can try recreating it with the feature enabled.

**Pull request merged**
